test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -165,6 +165,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.cloud.deployment_url (string)',
         'xpack.cloud.full_story.enabled (boolean)',
         'xpack.cloud.full_story.org_id (any)',
+        'xpack.cloud.full_story.eventTypesAllowlist (array)',
```

Comment:
__optional nit:__ would be great to have a comment with every newly added setting that would clarify what type of values it can contain, just for the future readers, but it's up to you.
```suggestion
        // Trust me, it's just an arbitrary string, no credentials, no IP addresses!
        'xpack.cloud.full_story.eventTypesAllowlist (array)',
```

Reply:
Sure! I'll add it in a follow up PR.

Reply:
https://github.com/elastic/kibana/pull/131527